index.html

```diff
@@ -19,12 +19,18 @@
   <link rel="stylesheet" href="css/simple-grid.css">
   <title>Sunburst Smorgasbord | a tool to describe your relationships.</title>
   <meta name="description" content="A customizable smorgasbord to describe your relationships.">
+  <script src="https://cdn.rawgit.com/eligrey/canvas-toBlob.js/f1a01896135ab378aa5c0118eadd81da55e698d8/canvas-toBlob.js"></script>
```

Comment:
Could you download these JS files and put them in the `js` folder instead? More of a slight preference than a must have though.